packages/terra-navigation-prompt/src/terra-dev-site/doc/navigation-prompt/implementation-guide.1/Part0-ReviewingExample.1.doc.mdx

```diff
@@ -13,68 +19,5 @@ The Form component renders a text input element and keeps the input's value in s
 
 If we enter text into Form 1's input, switch to Form 2, and then switch back to Form 1, we will notice our previously entered value is gone. This is not good, because that data in Form 1 could be *very* important to our users.
 
-```jsx
-import React, { useState } from 'react';
-import PropTypes from 'prop-types';
-
-const Form = ({ title }) => {
-  const [inputValue, setInputValue] = useState('');
-
-  return (
-    <div>
-      <p>{title}</p>
-      <input
-        type="text"
-        onChange={(event) => {
-          setInputValue(event.target.value);
-        }}
-        value={inputValue}
-      />
-      <button
-        type="button"
-        onClick={() => {
-          setInputValue('');
-        }}
-      >
-        Submit
-      </button>
-    </div>
-  );
-};
-
-Form.propTypes = {
-  title: PropTypes.string,
-};
-
-const FormSwitcher = () => {
-  const [activeForm, setActiveForm] = useState('Form 1');
-
-  function onSwitchForm(formKey) {
-    setActiveForm(formKey);
-  }
-
-  return (
-    <div>
-      <h2>Form Switcher</h2>
-      <p>The NavigationPrompt is not implemented, so no prompting occurs.</p>
-      <button
-        type="button"
-        disabled={activeForm === 'Form 1'}
-        onClick={onSwitchForm.bind(null, 'Form 1')}
-      >
-        Switch to Form 1
-      </button>
-      <button
-        type="button"
-        disabled={activeForm === 'Form 2'}
-        onClick={onSwitchForm.bind(null, 'Form 2')}
-      >
-        Switch to Form 2
-      </button>
-      <Form title={activeForm} key={activeForm} />
-    </div>
-  );
-};
-
-export default FormSwitcher;
-```
+## Example 
+<ImplementationGuidePart0 title='Part 0 - Reviewing the Initial FormSwitcher Implementation' />
```

Comment:
Since the code segment has been removed, we should add the toggle example code for this.

Reply:
Added